### DISCUSSION TITLE
How to set many file Old.log when rotation ?

### DISCUSSION THREAD

**Discussion by ngthien0402** [megahertz/electron-log#251] [Q&A]
Hi,
I had setting maxSize = 500kbs
And I want to rotation log file will have fileName.old.1.log, fileName.old.2.log ......
Is there any way to do that ?
Thanks

**Comment by megahertz** [comment]
Hi, you can implement your own behavior by overriding [archivelog](https://github.com/megahertz/electron-log/blob/master/docs/file.md#archivelog-oldlogfile-logfile--void)

**Reply by ngthien0402** [reply]
Thank you, I have resolved it
